subcommand/server-acl-init/command.go

```diff
@@ -82,6 +85,12 @@ func (c *Command) init() {
 		"Toggle for creating a token for a Connect mesh gateway")
 	c.flags.StringVar(&c.flagTimeout, "timeout", "10m",
 		"How long we'll try to bootstrap ACLs for before timing out, e.g. 1ms, 2s, 3m")
+	c.flags.StringVar(&c.flagConsulCACert, "consul-ca-cert", "",
+		"Path to the PEM-encoded CA certificate of the Consul cluster.")
+	c.flags.StringVar(&c.flagConsulTLSServerName, "consul-tls-server-name", "",
+		"The server name to set as the SNI header when sending HTTPS requests to Consul.")
+	c.flags.BoolVar(&c.flagUseHTTPS, "use-https", false,
```

Comment:
If we set `consul-ca-cert` or `consul-tls-server-name` with `use-https=false`, does it work? What about vice versa? Wondering if we should error out if not all the flags are set.

Reply:
Yeah, good question. If you set `use-https` but choose to add your CA to the system trusted CAs, then this should still work. If you set ca cert and tls server name, but don't enable https, it won't have any effect because the job will just use HTTP for all connections, so I don't want to error here.